lang/zh_CN.lang

```diff
@@ -68,22 +69,26 @@ screen.NETHER_B=玄武岩三角洲
 screen.END_COLOR=末地颜色
 
 screen.CGT=色彩分级与色调映射
-screen.EXPOSURE_CONFIG=更多曝光配置
-screen.COLOR_GRADING_CONFIG=色彩分级配置
+screen.EXPOSURE=更多曝光配置
+screen.COLOR_GRADING=色彩分级配置
 screen.CG_R=红色通道
 screen.CG_G=绿色通道
 screen.CG_B=蓝色通道
 screen.CG_T=色调
 
-screen.EXTRAS=附加项
+screen.ANTIALIASING=抗锯齿
 
-screen.ANIMATION=动画
+screen.EXTRAS=附加项
 
 #Settings
-option.ABOUT=BSL v8.2.09
-option.ABOUT.comment=作者：Capt Tatsu. bitslablab.com
+option.ABOUT=BSL v8.4
```

Comment:
please change the version number to v8.4.01.2

Reply:
changed